src/nsolid.h

```diff
@@ -583,6 +590,27 @@ NODE_EXTERN int OnUnblockedLoopHook(Cb&& cb, Data&&... data);
 template <typename Cb, typename... Data>
 NODE_EXTERN int OnConfigurationHook(Cb&& cb, Data&&... data);
 
+/**
+ * @brief The struct containing log info that's passed to the OnLogWriteHook
+ * callback.
+ */
+struct LogWriteInfo {
+  std::string info;
+  int level;
+};
```

Comment:
By looking at https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#log-and-event-record-definition, at the very least we should support:
- Timestamp
- TraceId / SpanId / TraceFlags if tracing is enabled.

We may add more later if we thing we need them.